Libs/DICOM/Widgets/ctkDICOMBrowser.cpp

```diff
@@ -140,7 +143,9 @@ class ctkDICOMBrowserPrivate: public Ui_ctkDICOMBrowser
   ctkFileDialog* ImportDialog;
   ctkDICOMMetadataDialog* MetadataDialog;
 
+#if QT_VERSION >= QT_VERSION_CHECK(5,0,0)
```

Comment:
The check should be done based on enabled/disabled features. We should avoid features disappearing silently based on detected Qt version.